src/Components/HomePage/AllBooks/AllBooks.jsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react"
+import { NavLink } from "react-router-dom";
+import BookListing from "./BookListing/BookListing"
+import AllBooks from "./AllBooks.css"
+
+function Allbooks() {
+    
+const [allBooks, setAllBooks] = useState([])
+
+//Contains the ID of the book that the user selects
+const[clickedBook, setClickedBook] = useState(null);
+
+    useEffect( function () {
+        fetch('https://book-swap-api.dev.io-academy.uk/api/books')
+        .then(function(res) {
+            return res.json()
+        })
+        .then(function (bookData) {
+            setAllBooks(bookData.data)
+            console.log(bookData.data)
+        })
+    }, [])
+    
+    return (
+        <>
+    
+            {<div className= "all-books">
```

Comment:
remove the extraspaces and the curly bracket in front of the <div>

Reply:
Yep, done and fixed up indents.